src/docs/exampleComponents/ComposedChart/BandedChart.js

```diff
@@ -0,0 +1,83 @@
+import React, { PureComponent } from 'react';
+import {
+  ComposedChart,
+  Line,
+  Area,
+  XAxis,
+  YAxis,
+  CartesianGrid,
+  Tooltip,
+  ResponsiveContainer,
+} from 'recharts';
+
+const data = [
+    {
+      name: "Page A",
+      a: [0, 0],
+      b: 0,
+    },
+    {
+      name: "Page B",
+      a: [50, 300],
+      b: 106,
+    },
+    {
+      name: "Page C",
+      a: [150, 423],
+    },
+    {
+      name: "Page D",
+      b: 312,
+    },
+    {
+      name: "Page E",
+      a: [367, 678],
+      b: 451,
+    },
+    {
+      name: "Page F",
+      a: [305, 821],
+      b: 623,
+    },
+  ];
+  
+  export default class Example extends PureComponent {
```

Comment:
Nit: this could be a function component without issue

Reply:
absolutely! I just kept consistency with other examples. But if no objections - I'm converting into a function!

Reply:
The other examples are very old for the most part 😅. Thanks!